instrumentation/netty/netty-3.8/src/main/java/io/opentelemetry/auto/instrumentation/netty/v3_8/server/NettyHttpServerTracer.java

```diff
@@ -67,6 +72,11 @@ protected String peerHostIP(final Channel channel) {
     return null;
   }
 
+  @Override
+  protected String flavor(Channel channel, HttpRequest request) {
+    return null; // TODO flavor is on the request, not the connection for this tracer
```

Comment:
does this work?
```suggestion
    return request.getProtocolVersion().toString();
```

Reply:
yep, I just missed this one when I added the request param.